wiki/Skin/en.md

```diff
@@ -1,10 +1,24 @@
----
-stub: true
----
-
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players use skins to allow for a more customised user experience. Most of the important UI elements and sounds in osu! can be "skinned". 
+
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since many skins hide certain visual details or offer gameplay elements focused on a certain playstyle like specific mod combinations.
+- In addition to gameplay focused changes, skins can also offer custom menus which commonly feature characters from various anime and games, as well as being based around certain design styles, like minimalism or techno.
+
+## How to Use Custom Skins
+
+Skins can either be [created from scratch](/wiki/Skinning), or downloaded as a `.osk` file and imported into the game. To import an `.osk` file simply open it with osu!, or drag-and-drop it into osu!. If a skin you download is in `.zip` or `.rar` format unzip it and move it into the `Skins` folder of your osu! installation. Make sure that you are moving the folder containing the `skin.ini` and not a folder that contains the folder.
+
+## Popular Skin Websites
```

Comment:
I don't feel "websites" is the right choice here, provided 3 of 5 links point to the forum threads
```suggestion
## Popular skin collections
```